tasks/abs.rb

```diff
@@ -62,30 +68,30 @@ def provision(platform, inventory_location, vars)
 
   raise 'Timeout: unable to get a 200 response in 10 minutes' if reply.code != '200'
```

Comment:
This should be raised to 1h for CI runs, and would make it in line with vmfloaty. For devs and local runs, we can use priority => 1 and skip the queue and the expectation is to wait 2-3 minutes if there are windows nodes (max is around 5 minutes) so it is a sound value for local dev runs.

Maybe have if priority 3 wait 1h, else 10 minutes?